third_party/pcsc-lite/naclport/common/src/google_smart_card_pcsc_lite_common/scard_structs_serialization.h

```diff
@@ -24,18 +24,14 @@
 // THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 // This file contains structure and function definitions that help to perform
-// (de)serialization of PC/SC-Lite API structures from/to Pepper values.
+// (de)serialization of PC/SC-Lite API structures from/to `Value`s.
 //
-// One of the reasons of defining separate intermediate structures is that the
-// original PC/SC API functions do not belong to the google_smart_card
```

Comment:
(This issue with namespaces isn't valid anymore due to the way how the new `value_conversion.h` is implemented, so I just  deleted this.)